packages/web/src/scss/components/TextField/_TextField.scss

```diff
@@ -149,43 +133,23 @@
     width: (2 * theme.$input-password-toggle-padding) + theme.$input-password-toggle-icon-size;
     height: 100%;
     border-radius: theme.$input-border-radius;
-    box-shadow: theme.$input-focus-shadow;
+    box-shadow: form-fields-theme.$input-focus-shadow;
 }
 
 .TextField__message,
 .TextField > [data-element='validator_message'] {
-    @include typography.generate(theme.$message-typography);
-
-    margin-top: theme.$spacing;
-    color: theme.$message-color-default;
+    @include form-fields-tools.message();
 }
 
-@each $variant-name, $variant-value in theme.$variants {
-    :is(.TextField--#{$variant-name}, .TextField.has-#{$variant-name}) > .TextField__input,
-    :is(.TextField--#{$variant-name}, .TextField.has-#{$variant-name})
-        .TextField__passwordToggle
-        > .TextField__input
-        ~ .TextField__passwordToggle__button::before {
-        border-color: map.get($variant-value, input-border-color);
```

Comment:
This could be changed to use a custom property:

```css
border-color: var(--input-border-color);
```

The value of the property would be assigned in the mixin below.

Reply:
https://github.com/lmc-eu/spirit-design-system/pull/536#discussion_r1023051860